storage-proto/Cargo.toml

```diff
@@ -12,6 +12,7 @@ edition = "2018"
 [dependencies]
 bincode = "1.3.3"
 bs58 = "0.4.0"
+enum-iterator = "0.7.0"
```

Comment:
This can go under `[dev-dependencies]` instead, right?

I think we'll also need to switch the type annotations to
```
#[cfg_attr(test, derive(enum_iterator::IntoEnumIterator))]
```

Reply:
Ah, good call, thanks!